### ISSUE TITLE
hocr-pdf printing Hebrew text in opposite direction in the generated pdf file

### ISSUE THREAD

**Issue by smijo149** [ocropus/hocr-tools#163]
The pdf file generated using `hocr-pdf` has Hebrew text printed in the opposite direction.

Steps I followed:
1. I used Google cloud vision to get the OCR 
2. Used gcv2hocr to generate hocr.
3. Used `hocr-pdf --savefile output.pdf actual-file.jpg` to generate pdf file.

The pdf file has Hebrew text inserted in it but in the reverse order.

### Actual image:

<img width="262" alt="Screen Shot 2021-02-01 at 6 48 35 PM" src="https://user-images.githubusercontent.com/3844756/106536559-53f41100-64be-11eb-93b2-a3576a917a00.png">

### This is how my hocr file looks:

<img width="623" alt="Screen Shot 2021-02-01 at 7 01 04 PM" src="https://user-images.githubusercontent.com/3844756/106537430-ed6ff280-64bf-11eb-8954-fed6da879428.png">

### Text in pdf file: (I have set text visibility mode to 0 so that the inserted text is visible)

<img width="265" alt="Screen Shot 2021-02-01 at 6 48 56 PM" src="https://user-images.githubusercontent.com/3844756/106536944-fad8ad00-64be-11eb-9ff4-c95ad4a4accf.png">

Hebrew is a right to left language so not sure if I have to pass any language or direction parameters to get this right.

**Comment by stweil** [comment]
I am afraid that `hocr-pdf` was never tested with RTL text. Using `bidi` like in https://github.com/tesseract-ocr/tesstrain/blob/master/generate_wordstr_box.py might fix that.

**Comment by smijo149** [comment]
Thanks! I will try it out and see if that works for me.

**Comment by joewiz** [comment]
@smijo149 Looks like you solved this. I wonder if the maintainers of hocr-tools would be interested in your PR?

**Comment by smijo149** [comment]
@joewiz Yeah I was able to solve the issue based on @stweil suggestion. I have opened a PR #165  if anyone is interested. Thanks!
